### ISSUE TITLE
Incorrectly auto-generating some paths

### ISSUE THREAD

**Issue by DenisSemionov** [ng-packagr/ng-packagr#1109]
## Type of Issue

```
[x] Bug Report
[ ] Feature Request
```

## Description

The issue is when I run the command ng build *package_name* everything (classes and etc.) builds fine except some paths in functions.

For example, it generates line

export declare const selectState: import (**"../../../AppFolder/ClientApp/node_modules/@ngrx/store/src/selector"**).foobar

when correct path would be

export declare const selectState: import (**"../../AppFolder/ClientApp/node_modules/@ngrx/store/src/selector"**).foobar

and everytime after I build the package I need to remove **`../`** in each path which is kind of annoying and takes a lot of time due to many changes made to the library.

Problem persists only with this node_modules/@ngrx/store/src/selector

#### How To Reproduce

I'm really not sure about this step. But let me try

1. Create selector
export const selectState = createFeatureSelector<ItemState>('item'); 

2.Export a const selector
export const getItemLoading = createSelector(
    selectState,
    (state: ItemState) => state.loading
);

3. Compile package

For me that's all, paths are generated incorrectly...

#### Expected Behaviour

All paths should be generated correctly.

#### Version Information

```
$ node_modules/.bin/ng-packagr --version
ng-packagr: ^4.2.0
@angular/*: ^6.1.9
typescript: 2.9.2
rxjs: ^6.3.3
node: 10.7.0
npm/yarn: 6.4.1/-

"@ngrx/core": "^1.2.0",
"@ngrx/effects": "^6.1.0",
"@ngrx/entity": "^6.1.0",
"@ngrx/router-store": "^6.1.0",
"@ngrx/store": "^6.1.0",
```

**Comment by DenisSemionov** [comment]
@alan-agius4 , Thank you. Unfortunately, we're using angular/flex-layout and some more libraries, so I cannot upgrade to newest typescript and angular 7.0.0rc version right now. Probably, the other solution would be to use facades if I am correct.

**Comment by DenisSemionov** [comment]
I have checked, this issue indeed fixed in TS 3.1. :)
